### ISSUE TITLE
Tarball checksum validation failed

### ISSUE THREAD

**Issue by diegomontoya** [netdata/netdata#10340]
Ubuntu 20.04
X86-64

```
[/tmp/netdata-kickstart-PfyFGgU46M]# curl -q -sSL --connect-timeout 10 --retry 3 --output /tmp/netdata-kickstart-PfyFGgU46M/sha256sum.txt https://storage.googleapis.com/netdata-nightlies/sha256sums.txt  OK  

[/tmp/netdata-kickstart-PfyFGgU46M]# curl -q -sSL --connect-timeout 10 --retry 3 --output /tmp/netdata-kickstart-PfyFGgU46M/netdata-latest.tar.gz https://storage.googleapis.com/netdata-nightlies/netdata-latest.tar.gz  OK  

 ABORTED  Tarball checksum validation failed. Stopping netdata installation and leaving tarball in /tmp/netdata-kickstart-PfyFGgU46M 
```

Exactly same as https://github.com/netdata/netdata/issues/8967

**Comment by OdysLam** [comment]
Hey @diegomontoya ,

Thanks for posting this issue. I think this is related to a problem we have identified with our updater. Please be patient and one of our engineers will come back with more information. Perhaps @Ferroin can provide more context?

**Comment by lamka02sk** [comment]
This should be a priority, the installer is broken completely 😞

**Comment by Ferroin** [comment]
Has anybody actually retried this? It’s a known issue if you try to update right around the same time that our nightly releases get published due to Google Cloud Storage not preemptively invalidating their edge caches when the build assets are updated (and we have no way to work around this), resulting in the files not matching on occasion when fetched around about 00:30-01:30 UTC.

If you are _still_ seeing issues despite testing outside of that time range, then I will be happy to look into them if you can provide more details about your specific setup, but I have been unable to reproduce any such issues locally on any of my test systems.

**Comment by OdysLam** [comment]
Since this is a recurring theme, perhaps @manos-saratsis @dim08 you are interested in tackling this from a product perspective. Perhaps a different error message?

**Comment by lamka02sk** [comment]
@Ferroin Today, I tried running the installer again and it worked. It looks like I hit the mentioned invalid cache window since I commented right around that time.

**Comment by manos-saratsis** [comment]
@OdysLam I think the message is fine. An idea for improvement would be to ask the user to redo the installation @Ferroin what's your thoughts on this.

**Comment by Ferroin** [comment]
Indeed, I think we could probably update the message to suggest retrying the install in about an hour. I could even code it to only display this around the time we typically publish nightly releases if we want.

**Comment by manos-saratsis** [comment]
@Ferroin great. Please always suggest to reinstall when the checksum fails.
